packages/core/src/Provider/StakePoolProvider/types/StakePoolProvider.ts

```diff
@@ -58,6 +58,10 @@ export interface StakePoolSearchResults {
 }
 
 export interface StakePoolStats {
+  averages: {
+    margin: number;
```

Comment:
Please document what the new values are, e.g.

```suggestion
    /** 
     * Within range [0; 1]
     */
    margin: Percent;
```